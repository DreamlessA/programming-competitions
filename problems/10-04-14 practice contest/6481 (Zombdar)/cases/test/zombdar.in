9

RUN;
END OF CASE
Zom
bies:
 5;Zombies: 1
;
No Zombies;
Zombies: 70;
RUN;
RUN;RUN;RU
END OF CASE
No
 Zombies;
No
 Zombies;
Zombies: 4;Z
ombies
: 14;
Zombies
: 60;
Zombies:
 100;

Zom
bies: 15;
RUN;
RUN;RUN;
R
END OF CASE

END OF CASE
END OF CASE
Zombies: 1
2;
RU
N;
END OF CASE
Z
o
m
b
i
e
s
END OF CASE
RUN;RUN;RUN;RUN;RUN;RUN;RUN;RUN;
END OF CASE
Zombies: 1;
No Zombies;
Zombies: 1;
No Zombies;
Zombies: 1;
No Zombies;
Zombies: 1;
No Zombies;
RUN
;
RUN
;
RUN
END OF CASE
