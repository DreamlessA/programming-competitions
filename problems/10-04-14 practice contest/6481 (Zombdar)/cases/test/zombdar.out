2: RUN;
3: Zombies: 5;
4: Zombies: 1;
5: No Zombies;
6: Zombies: 70;
7: RUN;
8: RUN;
8: RUN;
2: No Zombies;
4: No Zombies;
5: Zombies: 4;
7: Zombies: 14;
9: Zombies: 60;
11: Zombies: 100;
14: Zombies: 15;
15: RUN;
16: RUN;
16: RUN;
2: Zombies: 12;
4: RUN;
1: RUN;
1: RUN;
1: RUN;
1: RUN;
1: RUN;
1: RUN;
1: RUN;
1: RUN;
1: Zombies: 1;
2: No Zombies;
3: Zombies: 1;
4: No Zombies;
5: Zombies: 1;
6: No Zombies;
7: Zombies: 1;
8: No Zombies;
10: RUN;
12: RUN;
